internal/network_acl_context_test.go

```
@@ -0,0 +1,116 @@
+		t.Fatalf("input[network_acl] should contain the raw Network ACL map")
+	}
+	if networkAclMap["IsDefault"] != false {
+		t.Fatalf("network_acl.IsDefault = %v, want false", networkAclMap["IsDefault"])
+	}
+
+	contextMap, ok := input["nacl_context"].(map[string]interface{})
+	if !ok {
+		t.Fatalf("input[nacl_context] should be a map")
+	}
+
+	current := contextMap["current"].(map[string]interface{})
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)